src/applications/mhv-medical-records/components/CareSummaries/ProgressNoteDetails.jsx

```diff
@@ -115,27 +115,47 @@ ${record.note}`;
       />
       <DownloadingRecordsInfo allowTxtDownloads={allowTxtDownloads} />
 
-      <div className="test-details-container max-80">
+      <div className="test-details-container max-80" data-dd-privacy="mask">
         <h2>Details</h2>
-        <h3 className="vads-u-font-size--base vads-u-font-family--sans">
+        <h3
+          className="vads-u-font-size--base vads-u-font-family--sans"
+          data-dd-privacy="mask"
+        >
           Location
         </h3>
```

Comment:
You don't need to mask out static text. Only data that is loaded from the user's records. This goes for all the `<h3>`s here.